pkg/kapp/app/recorded_app.go

```diff
@@ -143,6 +217,17 @@ func (a *RecordedApp) mergeAppUpdates(cm *corev1.ConfigMap, labels map[string]st
 }
 
 func (a *RecordedApp) Exists() (bool, string, error) {
+	if a.isMigrationEnabled() {
```

Comment:
Looks like here we are making an assumption that once users enable `KAPP_FQ_CONFIGMAP_NAMES` env variable it will not disable it again. If this happens unwillingly he will unable to find his apps.

Reply:
yep that's correct, we have a set of manual steps they can take to downgrade if they choose that path. If they don't provide the env_var they will not be able to find their migrated apps. I think this is a reasonable behaviour, we don't want them to be able to get into a bad state with the migrated app if they forget the variable.

Reply:
okay got it. Sounds Good.